modules/installation-openstack-nfv-requirements.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-nfv-preparing
+
+:_content-type: PROCEDURE
```

Comment:
This looks more like a reference than a procedure.

Reply:
```suggestion
:_content-type: REFERENCE
```